src/trec_search.cc

```diff
@@ -156,15 +156,6 @@ int main(int argc, char **argv)
 			enquire.set_weighting_scheme(Xapian::TradWeight());
 		}
  	}
-	else if (config.use_weightingscheme("lmweight")) {
```

Comment:
We were not suppose to remove LMWeight Weighting scheme. Scheme has changed specifically one of the field was removed. We had to adjust the constructor